ui/app/AppLayouts/Chat/views/ContactsColumnView.qml

```diff
@@ -296,32 +296,34 @@ Item {
 
     Connections {
         target: root.store
+
         function onImportingCommunityStateChanged(communityId, state, errorMsg) {
+
+            const community = root.store.getCommunityDetailsAsJson(communityId)
             let title = ""
+            let subTitle = ""
             let loading = false
 
-            if (state === Constants.communityImported)
-            {
-                title = qsTr("Community imported")
-            }
-            else if (state === Constants.communityImportingInProgress)
+            switch (state)
             {
+            case Constants.communityImported:
+                title = qsTr("%1 community imported").arg(community.name);
+                break
+            case Constants.communityImportingInProgress:
                 title = qsTr("Importing community is in progress")
                 loading = true
-            }
-            else if (state === Constants.communityImportingError)
-            {
-                title = errorMsg
-            }
-
-            if(title == "")
-            {
-                console.error("unknown state while importing community: ", state)
+                break
+            case Constants.communityImportingError:
+                title = qsTr("%1 community importing error").arg(community.name)
```

Comment:
Should probably be rather someting like "Error impoting community 'NAME'" or "Failed to import community 'NAME'"